### PR TITLE
TM-863 ClickTale tag template

### PR DESCRIPTION
**Pull request by bua89**

### CODE REVIEW

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace SevenTag\Plugin\ClickTaleCustomTemplateBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class}
```
Comment (kallosz) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/Configuration.php]: remove default comments

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('seven_tag_plugin_click_tale_custom_template');
+
+        // Here you should define the parameters that are allowed to
+        // configure your bundle. See the documentation linked above for
```
Comment (kallosz) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/Configuration.php]: remove default comments

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/SevenTagPluginClickTaleCustomTemplateExtension.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
```
Comment (kallosz) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/DependencyInjection/SevenTagPluginClickTaleCustomTemplateExtension.php]: remove default comments

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Form/ClickTaleTemplateFormType.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+ * @package SevenTag\Plugin\ClickTaleCustomTemplateBundle\Form
+ */
+class ClickTaleTemplateFormType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('partition', 'text')
```
Comment (kallosz) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Form/ClickTaleTemplateFormType.php]: `->add('partition', 'text')`
should be in new line.

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Resources/public/js/stgClickTale.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    'Project GUID',
+                    'Set project GUID'
+                ])
+                .then(function(translations) {
+
+                    $templateProvider
+                        .add(TEMPLATE_ID, TEMPLATE_NAME)
+                        .addTextField({
+                            name: 'partition',
+                            label: translations['Partition'],
```
Comment (kallosz) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Resources/public/js/stgClickTale.js]: ```46:35  error  ["Partition"] is better written in dot notation  dot-notation```
JS coding-standards

File: src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Resources/public/views/helper.html [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-info">
+    How to get project details from <a href="http://wiki.clicktale.com/Article/How_to_get_project_details_in_Balkan" target="_blank">ClickTale</a>.
```
Comment (michalsikora) [inline src/SevenTag/Plugin/ClickTaleCustomTemplateBundle/Resources/public/views/helper.html]: missing translates

**Pull request merged**